src/hipercow/ui.py

```diff
@@ -0,0 +1,50 @@
+"""Some quality of life printing utilities, ported from R's cli."""
+
+from rich.console import Console
+
+console = Console()
+
+
+# for h1/h2 we can use "Rule" I think
+# https://rich.readthedocs.io/en/latest/console.html#rules
+#
+# Nested lists are harder, but bullets won't be that hard to get
+# sorted.
+#
+# Pluralisation is the other thing we might want.
+
+
+def alert_danger(text: str, indent: int = 0) -> None:
+    alert(":heavy_multiplication_x:", text, "bold red", indent=indent)
+
+
+def alert_success(text: str, indent: int = 0) -> None:
+    alert(":heavy_check_mark:", text, "bold green", indent=indent)
+
+
+def alert_warning(text: str, indent: int = 0) -> None:
+    alert("!", text, "bold orange", indent=indent)
+
+
+def alert_info(text: str, indent: int = 0) -> None:
+    alert("i", text, "bold cyan", indent=indent)
+
+
+def alert_see_also(
+    text: str, prefix: str = "For more information, see ", indent: int = 0
+) -> None:
+    alert(":books:", f"{prefix}{text}", indent=indent)
+
+
+def alert_arrow(text: str, indent: int = 0) -> None:
+    alert(":arrow_forward:", text, "bold yellow", indent=indent)
+
```

Comment:
Maybe for a later PR - windows command prompt, and also git bash on windows, both perform badly with the UTF-8 characters, replacing them with a square. Colours are good though - and squares look kind of bullet-pointy, so let's not have this hold things up.